subprojects/packagefiles/wpdk/patches/wpdk-0001-fixed-clang-compilation-warning.patch

```diff
@@ -0,0 +1,261 @@
+From 9fa031ce3564b3d6902b3cc7822325379eb9c017 Mon Sep 17 00:00:00 2001
```

Comment:
I think these could/should be upstreamed to SPDK/WPDK. Looks like good improvements.

- [ ] Upstream patch fixing compilation-warnings